src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.Blocking.cs

```diff
@@ -260,6 +260,8 @@ private static class BlockingConfig
         {
             public static readonly bool IsCooperativeBlockingEnabled =
                 AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.Blocking.CooperativeBlocking", true);
+            public static readonly bool IgnoreMemoryUsage =
+                AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.Blocking.IgnoreMemoryUsage", false);
```

Comment:
Do we expect this to be set by actual consumers, or we're doing this purely for testing purposes?

Reply:
There are other knobs that allow removing the limits for thread injection, so I figured this could be a reasonable knob to be able to set, especially on machines with relatively lower memory and where the heuristic may limit thread injection when the starvation may be a larger issue.